pkg-manager/plugin-commands-installation/test/update/interactive.ts

```diff
@@ -314,11 +321,13 @@ test('interactively update should ignore dependencies from the ignoreDependencie
             value: '',
           },
           {
-            name: chalk`micromatch                                                   3.0.0 ❯ 3.{yellowBright.bold 1.10}                `,
+            message: chalk`micromatch                                                   3.0.0 ❯ 3.{yellowBright.bold 1.10}                `,
             value: 'micromatch',
+            name: 'micromatch',
           },
         ],
         name: 'dependencies',
+        disabled: true,
```

Comment:
I don't think disabling it is a good idea. It is useful to be able to select all dependencies or all devDependencies.

Reply:
How about ignore "dependencies", "devDependencies" or "peerDependencies" in the result list?

The only concern is that there is indeed an npm package called "dependeneies", but the last update was 11 years ago and it's not much used ~

Reply:
I think the value of the item passed to the prompt can be anything, right? So instead of passing the dependency name as the value, we could pass an object with a boolean like "isDependency".

Or alternatively, we could print `[dependencies]` instead and then filter out `[dependencies]`. This would avoid collision with real packages in the registry. This is a more dirty workaround, if you do this, then add a comment to the code explaining why the labels need to be called like that.

Reply:
thanks, the second looks good to me ~